extension/manifest.json

```diff
@@ -6,7 +6,7 @@
     {
       "matches": ["*://*/*projects*/issues/*",
```

Comment:
Shouldn't
`"*://*/*projects*/issues/*",`
that be:
`"*://*/*projects/*/issues/*",`